keycloak/deploy/docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM jboss/keycloak-ha-postgres
+
+ADD docker-entrypoint.sh /opt/jboss/
```

Comment:
Also need to change the standalone-ha.xml during the build to set
1. database configuration
1. logging configuration 
1. jgroups cluster configuration.